examples/jlang-parser/src/j.pest

```diff
@@ -0,0 +1,25 @@
+program = _{ SOI ~ "\n"* ~ (stmt ~ "\n"+) * ~ stmt? ~ EOI }
+
+stmt = _{ expr }
+
+expr = { assgmtExpr | monadicExpr | dyadicExpr | string | terms }
+monadicExpr = { action ~ expr }
+dyadicExpr = { (monadicExpr | terms) ~ action ~ expr }
+
+assgmtExpr = { ident ~ "=:" ~ expr }
+terms = { term+ }
+term = _{ decimal | integer | ident | "(" ~ expr ~ ")" }
+
+action = { verb ~ adverb* }
+verb = { ">:" | "*:" | "-" | "%" | "#" | ">." |
```

Comment:
Could you please split this up similarly to the [JSON example](https://github.com/pest-parser/book/blob/master/examples/json-parser/src/json.pest#L20)?